modules/test/playwright/tests/portal-web/html/common/themes/bottomJS.spec.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../../../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../../../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../../../../fixtures/loginTest';
+import {masterPagesPagesTest} from '../../../../../fixtures/masterPagesPagesTest';
+import {pageEditorPagesTest} from '../../../../../fixtures/pageEditorPagesTest';
+import {pageSelectorPagesTest} from '../../../../../fixtures/pageSelectorPagesTest';
+import {pagesAdminPagesTest} from '../../../../../fixtures/pagesAdminPagesTest';
+import {systemSettingsPageTest} from '../../../../../fixtures/systemSettingsPageTest';
+import getRandomString from '../../../../../utils/getRandomString';
+import {waitForAlert} from '../../../../../utils/waitForAlert';
+
+const test = mergeTests(
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	isolatedSiteTest,
+	loginTest(),
+	masterPagesPagesTest,
+	pageEditorPagesTest,
+	pageSelectorPagesTest,
+	pagesAdminPagesTest,
+	systemSettingsPageTest
+);
+
+const sampleJS = 'window.onload=function(){console.log("customjssnippets")}';
```

Comment:
No need for this to be a global variable, it can be withing test.

Reply:
Made changes